src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionSimulation.scala

```diff
@@ -207,6 +207,42 @@ class IncomeTaxSubmissionSimulation extends PerformanceTestRunner {
     postUkTaxEmploymentPage(taxYearEOY)
   )
 
+  setup("individual-remove-employment", "Individual Remove Employment - Single Employment") withRequests(
+    getLoginPage,
+    postIndividualLoginPage("AA133742A"),
+    getStartPage(taxYearEOY),
+    getEOYOverviewPage(taxYearEOY),
+    getEmploymentSummaryPage(taxYearEOY),
+    getRemoveEmploymentPage(taxYearEOY,"00000000-0000-1000-8000-000000000004"),
+    postRemoveEmploymentPage(taxYearEOY, "00000000-0000-1000-8000-000000000004"),
+    getEmploymentSummaryPage(taxYearEOY) //TODO - change to EOY overview page once caching problem is resolved
+  )
+
+  setup("agent-remove-employment", "Agent Remove Employment - Multiple Employments") withRequests(
+    getLoginPage,
+    postAgentLoginPage("BB444444A"),
+    getInsertAdditionalParametersEndPoint("BB444444A"),
+    getStartPage(taxYearEOY),
+    getEOYOverviewPage(taxYearEOY),
+    getEmploymentSummaryPage(taxYearEOY),
+    getRemoveEmploymentPage(taxYearEOY,"00000000-5555-0000-0000-000000000001"),
+    postRemoveEmploymentPage(taxYearEOY, "00000000-5555-0000-0000-000000000001"),
+    getEmploymentSummaryPage(taxYearEOY)
+  )
+
+  setup("individual-employment", "Individual Multiple Employments - Check Employment Details EOY")withRequests(
```

Comment:
I'm debating whether this scenario is needed as their wont be a difference in the requests made between an individual with multiple enrolments and an agent with multiple enrolments 🤔

Reply:
I think because its named "individual-employment" its going to overwrite the original "individual-employment" scenario as well.

Reply:
name has been changed to "individual-employment-details"